lib/src/realm_object.dart

```diff
@@ -458,9 +496,9 @@ class DynamicRealmObject {
   /// Gets a list by the property name. If a generic type is specified, the property
   /// type will be validated against the type. Otherwise, a `List<Object>` will be
   /// returned.
-  List<T> getList<T extends Object?>(String name) {
+  RealmList<T> getList<T extends Object?>(String name) {
```

Comment:
I think it was a mistake to expose this as `List`. It may be moot since @nielsenko plans to remove this API, but it was useful for now.

Reply:
Yes, you loose the extension methods :-) ..